internal/node/node.go

```diff
@@ -59,6 +60,12 @@ type AutopilotConfig struct {
 
 type ShutdownFn = func(context.Context) error
 
+var mainnet *consensus.Network
+
+func init() {
+	mainnet, _ = chain.Mainnet()
```

Comment:
Can you add support for `testnet` as well? We're thinking about using it for the MIT hackathon in April.